browser-test/src/trusted_intermediary.test.ts

```diff
@@ -392,7 +392,7 @@ describe('Trusted intermediaries', () => {
     }
     await tiDashboard.createClient(client3)
 
-    await tiDashboard.searchByDateOfBirth(client3.dobDate)
+    await tiDashboard.searchByDateOfBirth('10', '12', '2021')
```

Comment:
Can a test case like date = 1956 - blank - blank be added too?

Reply:
Good idea, yes, I'll add one.  Thanks!

Reply:
Added.